geocoding_ios/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 2.2.0
+
+  * Updates `geocoding_platform_interface` to version 3.1.0.
+  * Updates iOS Minimal os version to 12.
```

Comment:
It would be good to mention this only relates to the example application
```suggestion
  * Updates iOS minimal os version of the example application to 12.
```